apps/docs/src/app/platform/component-docs/platform-upload-collection/platform-upload-collection-examples/platform-upload-collection-base-data-provider.ts

```diff
@@ -18,11 +18,16 @@ import {
 } from '@fundamental-ngx/platform';
 
 import { generateUploadCollectionItems } from './platform-upload-collection-items-generator';
+import { HttpClient } from '@angular/common/http';
 
 export class PlatformUploadCollectionDataProviderExample extends UploadCollectionDataProvider {
     items: UploadCollectionItem[] = generateUploadCollectionItems(50, 4, 2);
     private _cancelUploadNewFileIds: (string | number)[] = [];
```

Comment:
I know these are not part of the PR :D but I noticed they are missing the typedocs

Reply:
oh, it's the example! Sorry, you can ignore my comment. Should not review PRs before coffee.